homeassistant/components/launch_library/sensor.py

```diff
@@ -1,69 +1,75 @@
-"""A sensor platform that give you information about the next space launch."""
+"""Support for Launch Library sensors."""
 from __future__ import annotations
 
-from datetime import timedelta
-import logging
-
-from pylaunches import PyLaunches, PyLaunchesException
-import voluptuous as vol
-
-from homeassistant.components.sensor import PLATFORM_SCHEMA, SensorEntity
-from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME
-from homeassistant.helpers.aiohttp_client import async_get_clientsession
-import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
 
 from .const import (
     ATTR_AGENCY,
     ATTR_AGENCY_COUNTRY_CODE,
     ATTR_LAUNCH_TIME,
     ATTR_STREAM,
     ATTRIBUTION,
-    DEFAULT_NAME,
+    DOMAIN,
 )
 
-_LOGGER = logging.getLogger(__name__)
 
-SCAN_INTERVAL = timedelta(hours=1)
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the sensor platform."""
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string}
-)
+    coordinator = hass.data[DOMAIN]
 
+    sensors = [
+        NextLaunchSensor(coordinator, "Next launch"),
+    ]
```

Comment:
This can be done in a single go

```suggestion
    async_add_entities(
        NextLaunchSensor(coordinator, "Next launch"),
    )
```